data/json/items/magazine/nitrox.json

```diff
@@ -0,0 +1,36 @@
+[
+  {
+    "id": "nitrox_tank_metal",
+    "type": "MAGAZINE",
+    "name": "aluminum nitrox cylinder",
+    "description": "A standard 30 minute aluminum 2216 PSI cylinder for use with a breathing apparatus.",
+    "weight": 6000,
+    "volume": 7,
```

Comment:
Is it really has volume of only 1750 ml?

Reply:
Changed volume to 47.